icu4j/main/tests/core/src/com/ibm/icu/dev/test/rbbi/rbbitst.txt

```diff
@@ -39,7 +39,9 @@
 
 #   Temp debugging tests
 #
-
+<word>
```

Comment:
Note: there are two copies of the test file, one in ICU4C and one in ICU4J. They are supposed to be identical, kept in sync. This diff, and several others below, are from copying the ICU4C version over to ICU4J. They are OK.